docs/pages/component/props.mdx

```diff
@@ -272,6 +272,9 @@ Whether this video view should be focusable with a non-touch input device, eg. r
 Controls whether the player enters fullscreen on play.
 See [presentFullscreenPlayer](#presentfullscreenplayer) for details.
 
+> [!WARNING]
```

Comment:
why this change is part of the PR ? 🤔

Reply:
Media3 handles the fullscreen icon internally. When the user presses the fullscreen button, everything works correctly, and the fullscreen icon changes as expected. However, if we want to make the video fullscreen without using this button, we need to use the following logic `exoFullScreen.performClick()`. Using this approach with the fullscreen prop presents a problem because the icon does not change correctly. I think we can manage the fullscreen status using only this method `setFullScreen`.

Reply:
Can't we just update icon image @seyedmostafahasani ?

Reply:
@KrzysztofMoch
I've checked when I implement it but we can't update icon image manually.